packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -297,3 +307,41 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin {
 		},
 	};
 }
+
+/**
+ * Copies a potential `.dev.vars` target over to the worker's output directory
+ * (so that it can get picked up by `vite preview`)
+ *
+ * Note: This resolves the .dev.vars file path following the same logic
+ *       as `loadDotEnv` in `/packages/wrangler/src/config/index.ts`
+ *       the two need to be kept in sync
```

Comment:
this is not ideal.... ideally we should share the functionality 😓

we could do that by exposing some API from wrangler but that would feel a bit gnarly to me (since this seems like internal logic more than anything else to me)

in my opinion, one of the biggest strengths in having a monorepo setup would be to share code... so I would really consider having some internal package for shared logic such as this, in that way code could be shared very easily without needing to expose it publicly in any our packages

anyways that's quite overkill for something as simple thing, but anyways I wanted to mention it just because I think setting something like that can in general be very useful (and this need to share code does come up from time to time)

Reply:
We could certainly move this code into the wrangler package and reuse it (in the future).

Meanwhile this is pretty simple and intuitive to make it OK to have a copy here.

Can you put a comment in the Wrangler code to ensure that if that logic changes they should also update here?

Reply:
> Can you put a comment in the Wrangler code to ensure that if that logic changes they should also update here?

that slipped my mind! good suggestion! thanks 🙂

Reply:
done 🙂👍  https://github.com/cloudflare/workers-sdk/pull/7864/commits/744f80bfcfbba3315f3770e31a38776d64a863a7